package-lock.json

```diff
@@ -6405,34 +6405,6 @@
         "postcss": "^8.4"
       }
     },
-    "plugin-packs/postcss-preset-env/node_modules/@csstools/postcss-is-pseudo-class": {
-      "version": "1.0.1",
-      "resolved": "https://registry.npmjs.org/@csstools/postcss-is-pseudo-class/-/postcss-is-pseudo-class-1.0.1.tgz",
-      "integrity": "sha512-Gli5WXeq1+Zb65Kl5xes+R4LiKWIO6FKYRxsbxkZEuTVN7/OnSkm5arpb4xHWYFUbx3Lejr8dFb3ljLP2C9e1w==",
-      "dependencies": {
-        "postcss-selector-parser": "^6.0.8"
-      },
-      "engines": {
-        "node": "^12 || ^14 || >=16"
-      },
-      "peerDependencies": {
-        "postcss": "^8.3"
-      }
-    },
-    "plugin-packs/postcss-preset-env/node_modules/postcss-custom-properties": {
-      "version": "12.1.3",
-      "resolved": "https://registry.npmjs.org/postcss-custom-properties/-/postcss-custom-properties-12.1.3.tgz",
-      "integrity": "sha512-rtu3otIeY532PnEuuBrIIe+N+pcdbX/7JMZfrcL09wc78YayrHw5E8UkDfvnlOhEUrI4ptCuzXQfj+Or6spbGA==",
-      "dependencies": {
-        "postcss-value-parser": "^4.2.0"
-      },
-      "engines": {
-        "node": "^12 || ^14 || >=16"
-      },
-      "peerDependencies": {
-        "postcss": "^8.3"
-      }
-    },
```

Comment:
This was causing issues as it made `npm install` and `npm ci` to add the published version in `postcss-preset-env`.

I did a reset of the lock file to resolve this.